lib/features/contact/presentation/contact_bindings.dart

```diff
@@ -8,4 +8,8 @@ class ContactBindings extends Bindings {
   void dependencies() {
     Get.lazyPut(() => ContactController());
   }
+
+  void dispose() {
+    Get.delete<ContactController>();
```

Comment:
Since we are not calling the view directly by navigating in `Pages`, the `controller` attached to the view will not automatically cancel. So we need to clean it up.